### PR TITLE
refactor(*): Remove duplicate execution details templates

### PR DESCRIPTION
**Pull request by jrsquared**
Remove a lot of the duplicate `*ExecutionDetails.html` templates and provide them by default in core.

Fairly confident I didn't break other providers. Fairly.

### CODE REVIEW

File: app/scripts/modules/core/src/pipeline/config/stages/findAmi/findAmiStage.module.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { module } from 'angular';
+
+import { FIND_AMI_STAGE } from './findAmiStage';
+
+export const FIND_AMI_STAGE_MODULE = 'spinnaker.core.pipeline.stage.findAmi';
+
+module(FIND_AMI_STAGE_MODULE, [
+  FIND_AMI_STAGE,
+]);
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/pipeline/config/stages/findAmi/findAmiStage.module.ts]: I'd skip this file/module and just import the `FIND_AMI_STAGE` wherever you're importing `FIND_AMI_STAGE_MODULE`.

File: app/scripts/modules/core/src/pipeline/config/stages/resizeAsg/resizeAsgStage.module.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { module } from 'angular';
+
+import { RESIZE_ASG_STAGE } from './resizeAsgStage';
+
+export const RESIZE_ASG_STAGE_MODULE = 'spinnaker.core.pipeline.stage.resizeAsg';
+
+module(RESIZE_ASG_STAGE_MODULE, [
+  RESIZE_ASG_STAGE,
+]);
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/pipeline/config/stages/resizeAsg/resizeAsgStage.module.ts]: same as above (my opinion)

File: app/scripts/modules/core/src/pipeline/config/stages/findAmi/findAmiStage.module.ts (hunk 1 above)
Comment (jrsquared) [reply app/scripts/modules/core/src/pipeline/config/stages/findAmi/findAmiStage.module.ts]: There's a few of these... I'll clean them up.

File: app/scripts/modules/core/src/pipeline/config/stages/scaleDownCluster/scaleDownClusterExecutionDetails.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+          <div class="col-md-12">
+              <div class="well alert alert-info">
+                  <!-- TODO: Move this to config -->
+                  <a target="_blank" href="{{stage.context.execution.logs}}">
+                      View Execution Logs
+                  </a>
+              </div>
+          </div>
+      </div>
+  </div>
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/pipeline/config/stages/scaleDownCluster/scaleDownClusterExecutionDetails.html]: while you're in here, might as well fix the indent (4 spaces => 2 spaces)

**Comment by anotherchrisberry** [comment]
Some little formatting things, but A+ otherwise

**Comment by jrsquared** [comment]
here goes nothing.

**Pull request merged**